internal/cmd/schema-registry/schema_utilities.go

```diff
@@ -24,15 +24,15 @@ type registerSchemaResponse struct {
 }
 
 type RegisterSchemaConfigs struct {
-	SchemaDir   string
-	Subject     string
-	ValueFormat string
-	SchemaType  string
-	SchemaPath  string
-	Refs        []srsdk.SchemaReference
-	Metadata    *srsdk.Metadata
-	Ruleset     *srsdk.RuleSet
-	Normalize   bool
+	SchemaDir  string
+	Subject    string
+	Format     string
```

Comment:
a little confused here: does this format refer to key or value?

Reply:
Either! It's used for both.